sql/moz-fx-data-shared-prod/fenix_derived/new_profile_activation_v1/checks.sql

```diff
@@ -0,0 +1,9 @@
+#fail
+{{ is_unique("client_id", "submission_date = @submission_date") }}
```

Comment:
What is the expected grain here? Isn't it a single entry per client in the entire table? If so, maybe we should remove the where condition here?

Reply:
Good point - it is, but we still have dupes in firefox_android_clients and thus this table. This will fail every day, until we resolve https://bugzilla.mozilla.org/show_bug.cgi?id=1775137